packages/nx-plugin/project.json

```diff
@@ -40,6 +40,14 @@
       "command": "nx g ./dist/packages/nx-plugin:init",
       "dependsOn": ["build"]
     },
+    "exec:configuration": {
+      "command": "nx g ./dist/packages/nx-plugin:configuration",
+      "dependsOn": ["build"]
+    },
```

Comment:
We don't need a custom command for this, you can run the local Nx plugin with `nx g @code-pushup/nx-plugin:configuration ...`.

```suggestion
```

![image](https://github.com/code-pushup/cli/assets/34691111/03eccbaf-8316-4b11-96f3-771157ce0c3c)

Reply:
added docs and removes targets